src/content_types/all.nim

```diff
@@ -56,6 +56,8 @@ proc initTextFrame*(text: string): TextFrame =
   result = TextFrame(encoding: ord(Utf8), text: text)
 
 
+
+
```

Comment:
Is there a Linter that works well for nim? I've been looking for one

Reply:
There is nph (https://github.com/arnetheduck/nph) used in nwaku, seems fine. Let's try it.